src/oiiotool/oiiotool.cpp

```diff
@@ -5000,7 +5015,7 @@ OIIOTOOL_OP(point, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
 
 
 // --text
-OIIOTOOL_OP(text, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
+OIIOTOOL_INPLACE_OP(text, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
     img[0]->copy(*img[1]);
```

Comment:
I think you forgot to remove the copy here (like you've done in the 3 ops above)

Reply:
(for consistency)

Reply:
you're right, I will fix